pgmpy/tests/test_models/test_DynamicBayesianNetwork.py

```diff
@@ -140,6 +140,33 @@ def test_moralize(self):
                               [('D', 1), ('G', 1)], [('D', 1), ('I', 1)],
                               [('G', 0), ('I', 0)], [('G', 1), ('I', 1)]])
 
+    def test_copy(self):
+        self.network.add_edges_from(
+            [(('D', 0), ('G', 0)), (('I', 0), ('G', 0)), (('D', 0), ('D', 1)), (('I', 0), ('I', 1))])
+        copy = self.network.copy()
+        self.assertIsInstance(copy, DynamicBayesianNetwork)
+        self.assertListEqual(sorted(self.network.nodes()), sorted(copy.nodes()))
+        self.assertListEqual(sorted(self.network.edges()), sorted(copy.edges()))
+        self.assertListEqual(self.network.get_cpds(), copy.get_cpds())
+        self.assertListEqual(sorted(self.network.get_intra_edges()), sorted(copy.get_intra_edges()))
+        self.assertListEqual(sorted(self.network.get_inter_edges()), sorted(copy.get_inter_edges()))
+        self.assertListEqual(sorted(self.network.get_slice_nodes()), sorted(copy.get_slice_nodes()))
+        self.network.add_cpds(self.grade_cpd, self.d_i_cpd)
+        copy.add_cpds(self.diff_cpd, self.intel_cpd)
+        self.network.add_node('A')
+        copy.add_node('Z')
+        self.network.add_edge(('A', 0), ('D', 0))
+        copy.add_edge(('Z', 0), ('D', 0))
+        self.assertNotEqual(sorted(self.network.nodes()), sorted(copy.nodes()))
+        self.assertNotEqual(sorted(self.network.edges()), sorted(copy.edges()))
+        self.assertNotEqual(self.network.get_cpds(), copy.get_cpds())
+        self.assertNotEqual(sorted(self.network.get_intra_edges()), sorted(copy.get_intra_edges()))
+        self.assertListEqual(sorted(self.network.get_inter_edges()), sorted(copy.get_inter_edges()))
+        self.assertNotEqual(sorted(self.network.get_slice_nodes()), sorted(copy.get_slice_nodes()))
+        self.network.add_edge(('A', 0), ('D', 1))
+        copy.add_edge(('Z', 0), ('D', 1))
+        self.assertNotEqual(sorted(self.network.get_inter_edges()), sorted(copy.get_inter_edges()))
+
```

Comment:
@khalibartan  I actually don't know what method to use to edit CPD values of an existent network, could you tell me how to do this ?